pandas/core/ops.py

```diff
@@ -439,6 +461,18 @@ def f(x):
         self.lvalues = lvalues
         self.rvalues = rvalues
 
+
+    def _is_offset(self, arr_or_obj):
+        """ check if obj or first element of list-like is DateOffset """
+        if isinstance(arr_or_obj, pd.DateOffset):
+            return True
+        elif isinstance(arr_or_obj, pd.Series):
+            return isinstance(arr_or_obj.values[0], pd.DateOffset)
+        elif is_list_like(arr_or_obj):
```

Comment:
Changed this, added a test for `Series`.  Adding an array of offsets to a `DatetimeIndex` still fails, which I think is consistent with other `Index` behavior.